.github/workflows/reusable_codeql.yml

```diff
@@ -0,0 +1,48 @@
+name: "CodeQL"
+
+on:
+  workflow_call:
+    inputs:
+        languages:
+          description: "Language for CodeQL code check. Supported languages are: 'python', 'typescript', 'javascript' 'go', 'java', 'cpp', 'csharp', 'ruby'."
+          required: false
+          type: string
+          default: "['python']"
```

Comment:
Is there any harm in broadening this to include other languages? I know some submitters use languages other than python (for instance, the after-effects submitter also uses JS)

Reply:
Each repository will pass in their own languages they need run, python is just what I have as the default as most of our repositories use it.